### PR TITLE
added html file and css

### PR DESCRIPTION
**Pull request by mandelatuks**
added HTML
added CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+<div id="page-wrapper">
+  <header id="header">
+    <div class="logo">
+      <img
+        id="header-img"
+        src="images/metamask.png"
+      />
+    </div>
+
+    <nav id="nav-bar">
```
Comment (mandelatuks) [inline index.html:20]: in order to have a nice clean repo, it is not recommended to leave so many empty lines.
  .please go ahead and remove the empty lines in the HTML file and you can keep some only if there are separators between the section like head and body.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      </div>
+      <div class="grid">
+        
+        <div class="desc">
+          <h2>Pitbull</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia, quas.
+          </p>
+        </div>
+      </div>
```
Comment (mandelatuks) [inline index.html:73]: for semantic purposes please don't use too many div rather try to use sections, headers, kindly go ahead and change the main div here to section tags or main tags and change that to an article tag.

**Review by mandelatuks — COMMENTED** [review]
**To Highlight**
     . there were linters errors ❌ 
      . Descriptive pull request ❌ 
      . Descriptive commit messages ❌ 
      . READ file ❌ 

**Required Changes ♻️**
 kindly check the comments under the review.
        .please use the proper GitHub flow
        .please work on your README FILE so that it looks profession.
        . please work on your linters errors.
        . please use descrip[tive pull request and descriptive commit message
        

**Optional suggestions**
  
   -kindly in your CSS file try to use more classes than IDs 

  Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html:73]: Great job 👍 identifying the problem but you could write this in a more polite manner as `" Kindly make sure to use section tags, headers tags, for semantic purposes please avoid using too many div tags. Make sure to change the main div here to section tags or main tags ."` Please make sure you don't use the word `don't` as it hypers the student. Make sure to use words such as `kindly`, `make sure` with a proper sentence to make the student work tension free.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @mandelatuks, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job 👍  in identifying problems in the project.

## Required Changes ♻️
- Kindly make sure you mention that there was no linters setup for the project. You have mentioned there should be no linter errors. 
- Kindly note that Under highlights you cant mention what is not in the project. You have to only mention whats good in the project.
- Kindly avoid use of the ❌ symbol as it triggers the student. Use happy emojis as much as possible.
- _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mandelatuks) [inline index.html:73]: Kindly make sure to use section tags, headers tags, for semantic purposes please avoid using too many div tags. Make sure to change the main div here to section tags or main tags.

**Review by mandelatuks — COMMENTED** [review]
**Required Changes ♻️**

-  Kindly make sure to set up linters for this project as this will help you write standard code.
- kindly check the comment under the review.

**Optional suggestions**

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

 - make sure to use more classes than ids when you are styling your main page.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@mandelatuks ] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by mandelatuks** [comment]
@sja-thedude

**Review by Mre55 — APPROVED** [review]
**Hi @mandelatuks,**

# Status: APPROVED 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**